ohsome/clients.py

```diff
@@ -52,10 +52,11 @@ def __init__(
         else:
             self._base_api_url = OHSOME_BASE_API_URL
         self._cache = cache or []
-        if user_agent is not None:
-            self.user_agent = user_agent
-        else:
-            self.user_agent = "ohsome-py/{0}".format(OHSOME_VERSION)
+
+        if user_agent is None:
+            user_agent = "ohsome-py"
```

Comment:
Maybe make the default that nothing is appended to the default `ohsome-py/{OHSOME_VERSION}`